.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    groups:
+      github-actions:
+        patterns:
+          - "*"
```

Comment:
Not needed, there is no group to handle.

Reply:
This configures Dependabot to send a single PR updating all Actions with new versions, instead of one PR per Action (see [this PR on my fork](https://github.com/pnacht/moby/pull/1) for an example).

Admittedly, given the change to a daily schedule and use of major-version pins, that may be a difference without distinction.

But just to confirm, are you okay with that? If so, I'll happily remove the groups.